frontend_app_thread/src/container/Thread.jsx

```diff
@@ -418,12 +422,25 @@ export class Thread extends React.Component {
           i18n={i18n}
         >
           <div>
-            {state.showRefreshWarning && (
-              <RefreshWarningMessage
-                tooltip={this.props.t('The content has been modified by {{author}}', { author: state.editionAuthor, interpolation: { escapeValue: false } })}
-                onClickRefresh={this.handleClickRefresh}
+            <ToolBar>
+              <FavoriteButton
```

Comment:
If you decide to make a component, don't forget here

Reply:
See above.